src/components/announcer/Announcer.js

```diff
@@ -6,157 +6,110 @@
-import Snackbar from "@material-ui/core/Snackbar";
-import SnackbarContent from "@material-ui/core/SnackbarContent";
-import WarningIcon from "@material-ui/icons/Warning";
-
-import { withStyles } from "@material-ui/core/styles";
-
-const variantIcon = {
-    success: CheckCircleIcon,
-    warning: WarningIcon,
-    error: ErrorIcon,
-    info: InfoIcon,
-};
+import { Snackbar, useTheme, Typography } from "@material-ui/core";
+import Alert from "@material-ui/lab/Alert";
+
+function getTextColor(theme, variant) {
+    let color;
+    switch (variant) {
+        case Error:
+            color = theme.palette.error.contrastText;
+            break;
+        case SUCCESS:
+            color = theme.palette.success.contrastText;
+            break;
+        case WARNING:
+            color = theme.palette.warning.contrastText;
+            break;
+        default:
+            color = theme.palette.info.contrastText;
+    }
 
-const styles1 = (theme) => ({
-    success: {
-        backgroundColor: palette.darkGreen,
-    },
-    error: {
-        backgroundColor: palette.red,
-    },
-    info: {
-        backgroundColor: palette.blue,
-    },
-    warning: {
-        backgroundColor: palette.orange,
-    },
-    icon: {
-        fontSize: 20,
-    },
-    iconVariant: {
-        opacity: 0.9,
-        marginRight: theme.spacing(1),
-    },
-    message: {
-        display: "flex",
-        alignItems: "center",
-    },
-    margin: {
-        margin: theme.spacing(1),
-    },
-});
+    return color;
+}
 
 function MySnackbarContent(props) {
-    const { classes, message, onClose, variant, customAction } = props;
-    const Icon = variantIcon[variant];
-
+    const { message, onClose, variant, action } = props;
+    const theme = useTheme();
     return (
-        <SnackbarContent
-            className={classNames(classes[variant], classes.margin)}
-            aria-describedby="client-snackbar"
-            message={
-                <span id="client-snackbar" className={classes.message}>
-                    <Icon
-                        className={classNames(
-                            classes.icon,
-                            classes.iconVariant
-                        )}
-                    />
-                    {message}
-                </span>
-            }
-            action={[
-                customAction ? customAction() : null,
-                <IconButton
-                    key="close"
-                    aria-label="Close"
-                    color="inherit"
-                    className={classes.close}
-                    onClick={() => {
-                        console.log("close");
-                        onClose();
-                    }}
-                >
-                    <CloseIcon className={classes.icon} />
-                </IconButton>,
-            ]}
-        />
+        <Alert
+            elevation={6}
+            variant="filled"
+            severity={variant}
```

Comment:
Having this prop immediately after the `variant` prop kind of confused me.  Maybe we could do something like
```
const { message, onClose, variant: severity, action } = props;
```